pkg/images/images.go

```diff
@@ -51,7 +51,7 @@ func GetImagePath(repo, version, image, component string, imageOverrides map[str
 }
 
 func GetPullPolicy(version, override string) v1.PullPolicy {
-	if len(override) > 0 {
+	if len(override) > 0 && strings.Contains(override, ":") {
```

Comment:
Just curious - did you observe an actual panic in a running system or did you just notice this on code inspection?

```suggestion
        tags := strings.Split(override, ":")
	if len(tags) > 1 {
	    return getPullPolicy(tags[1])
	}
	
	return getPullPolicy(version)
```

Reply:
Yes,I have observe an actual panic in my running system,like the below image
<img width="902" alt="微信截图_20220908090401" src="https://user-images.githubusercontent.com/28950491/189011620-ef1736a7-d73d-4428-8e03-45019b82e03c.png">